readme.txt

```diff
@@ -32,5 +32,8 @@ There are two connection types: `internal` and `external`.
 
 == Upgrade Notice ==
 
+= 2.0.5 =
+**Distributor now requires WordPress 6.3 or later.**
```

Comment:
Probably could have bumped to 6.4 as minimum with 6.6 being the tested-up-to now.